Code/GraphMol/testChirality.cpp

```diff
@@ -2430,6 +2430,59 @@ void testAssignStereochemistryFrom3D() {
   BOOST_LOG(rdInfoLog) << "done" << std::endl;
 }
 
+void testDoubleBondStereoInRings() {
+  BOOST_LOG(rdInfoLog) << "-------------------------------------" << std::endl;
+  BOOST_LOG(rdInfoLog) << "Testing double bond stereochemistry in rings" << std::endl;
+  std::string pathName = getenv("RDBASE");
+  pathName += "/Code/GraphMol/test_data/";
+  {
+    RWMol *m = MolFileToMol(pathName + "cyclohexene_3D.mol", true, false); // don't remove Hs
+    MolOps::detectBondStereochemistry(*m);
+    MolOps::assignChiralTypesFrom3D(*m);
+    MolOps::assignStereochemistry(*m, true, true);
+    for (RWMol::BondIterator bondIt = m->beginBonds(); bondIt != m->endBonds(); ++bondIt) {
+      if ((*bondIt)->getBondType() == Bond::DOUBLE)
+        TEST_ASSERT((*bondIt)->getStereo() == Bond::STEREONONE);
+    }
+    delete m;
+  }
+  {
+    RWMol *m = MolFileToMol(pathName + "CHEMBL501674_3D.mol", true, false); // don't remove Hs
+    MolOps::detectBondStereochemistry(*m);
+    MolOps::assignChiralTypesFrom3D(*m);
+    MolOps::assignStereochemistry(*m, true, true);
+    for (RWMol::BondIterator bondIt = m->beginBonds(); bondIt != m->endBonds(); ++bondIt) {
```

Comment:
Since you know which atoms this bond is between, why not just use `getBondBetweenAtoms()` instead of this loop?